Tests/CasePathsMacrosTests/CasePathableMacroTests.swift

```diff
@@ -1209,10 +1209,52 @@ final class CasePathableMacroTests: XCTestCase {
         }
         public static var allCasePaths: AllCasePaths { AllCasePaths() }
       }
-
+      
       extension Foo: CasePaths.CasePathable, CasePaths.CasePathIterable {
       }
       """#
     }
   }
+
+  func testElementGeneric() {
+    assertMacro {
+      """
+      @CasePathable enum Action<Element> {
+        case element(Element)
+      }
+      """
+    } expansion: {
+      #"""
+      enum Action<Element> {
+        case element(Element)
+
+        public typealias _$Element = Element
```

Comment:
I don't get why this expansion has no indentation.

Reply:
Maybe it needs to be its own decl syntax line in the array above?